docs/qa.md

```diff
@@ -4,7 +4,7 @@
 
 All tests should be ran primarily on the supported platform:
 
-* Kaby Lake NUC ([details here](https://github.com/projectacrn/acrn-hypervisor/blob/master/doc/hardware.rst#intel-kaby-lake-nuc))
+* Kaby Lake NUC ([details here](https://projectacrn.github.io/latest/reference/hardware.html#verified-platforms-according-to-acrn-usage))
```

Comment:
It might be better to refer to v1.2 documentation instead as ACRN is actively being development.